src/Token/Parser.php

```diff
@@ -26,7 +26,7 @@ public function __construct(Decoder $decoder)
         $this->decoder = $decoder;
     }
 
-    public function parse(string $jwt): TokenInterface
+    public function parse(string $jwt): Plain
```

Comment:
This has already been discussed:

1. https://github.com/lcobucci/jwt/pull/758
2. https://github.com/lcobucci/jwt/issues/775
3. https://github.com/lcobucci/jwt/issues/786

Reply:
Thanks for pointing me towards this previous work. I took inspiration from your approach in `JwtFacade::parse()` and extended `Parser` itself with a method that expects to only handle unencrypted tokens.